WDL2CWL_example/samtoolsDocker/samtoolsView.cwl

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Does one of the tool descriptions in https://github.com/common-workflow-library/bio-cwl-tools/tree/release/samtools work for you?